packages/smart-wallet/src/walletFactory.js

```diff
@@ -14,6 +14,11 @@ import { E } from '@endo/far';
 import { makeSmartWallet } from './smartWallet.js';
 import { shape } from './typeGuards.js';
 
+// Ambient types. Needed only for dev but this does a runtime import.
+import '@agoric/vats/exported.js';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import '@agoric/swingset-vat/src/types-ambient.js';
```

Comment:
I don't suppose we can avoid relying on ambient types for `swingset-vat`. I know it does re-export most types explicitly at the top level.